Source/DafnyCore/AST/TopLevelDeclarations.cs

```diff
@@ -938,8 +938,8 @@ public class ModuleDefinition : INode, IDeclarationOrUsage, INamedRegion, IAttri
   public override IEnumerable<INode> Children => (Attributes != null ? new List<INode> { Attributes } : Enumerable.Empty<INode>()).Concat(TopLevelDecls);
 }
 
-public class DefaultModuleDecl : ModuleDefinition {
-  public DefaultModuleDecl()
+public class DefaultModuleDef : ModuleDefinition {
```

Comment:
Ahhhhhh I was wondering why I was always confused by module declarations and module definitions.... you nailed it !